### DISCUSSION TITLE
Unable to install

### DISCUSSION THREAD

**Discussion by cyang1993** [derekantrican/GAS-ICS-Sync#328] [Q&A]
Hi - I can't get install the app, whenever I sign in to review the permissions/authorize it I get the below message with no workaround. Can you please help?

![image](https://user-images.githubusercontent.com/15350206/220751894-6e465b46-4586-410f-8547-fe8fc5d1c2fc.png)

**Comment by derekantrican** [comment]
Did you make a copy of the project with all the files? Can you share a larger screenshot with all your files?

**Comment by cyang1993** [comment]
Hi - I did make a copy with all the files, but maybe I am missing something? Here is the screenshot:

![image](https://user-images.githubusercontent.com/15350206/221004590-6bdb80fe-d7f9-4837-bf31-5acb3a08caa4.png)

**Comment by derekantrican** [comment]
Interesting. So the when you run the "Install" function on your own copy it's saying "This app is blocked"? That's crazy that Google is blocking your own script. Are you using a special Google account for work (or some other organization) or is this just a personal Google Account?

There's some discussion here: https://stackoverflow.com/questions/67898285/how-to-resolve-this-app-is-blocked-error-for-shared-google-apps-script-library with simple solutions being "just use a different account" (which might be frustrating if all of your calendars are on this one) or to create a Google Cloud project.

**Comment by derekantrican** [comment]
This issue is pretty old at this point - @cyang1993 are you still impacted by this?

**Comment by HelloRussian1** [comment]
Hello, 

I have the same issue please help. I was able to do this last week or two weeks ago but now it is blocking me on all my accounts and device. :(

**Reply by derekantrican** [reply]
Can you please create a new discussion (https://github.com/derekantrican/GAS-ICS-Sync/discussions) with some more information? Screenshots, how you set up the script, etc would be helpful

**Comment by zvaligura** [comment]
I'm getting this same issue as well. I have been running this for years on my google account (thanks for the great work on this by the way).

However, I'm setting it up on my wife's google account and getting the blocked error mentioned here. I've copy and set it up a few different times to troubleshoot but her account is giving me issues I can't diagnose.

**Comment by zvaligura** [comment]
Well set up GCP on my wifes account as a free user and followed these instructions... and now it works. Go figure

https://aimanfikri.com/2022/05/09/this-app-is-blocked-error-on-google-apps-script-solution/

**Comment by HelloRussian1** [comment]
It worked for me after I manually setup and ran the code.
